addOns/soap/src/test/java/org/zaproxy/zap/extension/soap/WSDLFilePassiveScanRuleTestCase.java

```diff
@@ -61,13 +71,24 @@ public void isWsdlTest()
     }
 
     @Test
-    public void isXmlContentType() throws IOException {
+    public void ShouldNotAlertWithJustXmlContentType() throws IOException {
```

Comment:
The leading S should be in lowercase, same for the other test.

Reply:
You could also add a third test using `Sample#setSoapVersionTwoRequest()` for `application/soap+xml`.

Reply:
You could also add a third test for the `application/wsdl+xml` content type...